Assets/HoloToolkit/Input/Scripts/InputSources/InteractionInputSource.cs

```diff
@@ -147,168 +149,206 @@ public void ResetUpdatedBooleans()
 
-            gestureRecognizer.SetRecognizableGestures(GestureSettings.Tap |
+            GestureRecognizer.SetRecognizableGestures(GestureSettings.Tap |
                                                       GestureSettings.ManipulationTranslate |
                                                       GestureSettings.Hold);
 
             // We need a separate gesture recognizer for navigation, since it isn't compatible with manipulation
-            navigationGestureRecognizer = new GestureRecognizer();
+            NavigationGestureRecognizer = new GestureRecognizer();
 
-            navigationGestureRecognizer.NavigationStarted += NavigationGestureRecognizer_NavigationStarted;
-            navigationGestureRecognizer.NavigationUpdated += NavigationGestureRecognizer_NavigationUpdated;
-            navigationGestureRecognizer.NavigationCompleted += NavigationGestureRecognizer_NavigationCompleted;
-            navigationGestureRecognizer.NavigationCanceled += NavigationGestureRecognizer_NavigationCanceled;
+            NavigationGestureRecognizer.NavigationStarted += NavigationGestureRecognizer_NavigationStarted;
+            NavigationGestureRecognizer.NavigationUpdated += NavigationGestureRecognizer_NavigationUpdated;
+            NavigationGestureRecognizer.NavigationCompleted += NavigationGestureRecognizer_NavigationCompleted;
+            NavigationGestureRecognizer.NavigationCanceled += NavigationGestureRecognizer_NavigationCanceled;
 
             if (UseRailsNavigation)
             {
-                navigationGestureRecognizer.SetRecognizableGestures(GestureSettings.NavigationRailsX |
+                NavigationGestureRecognizer.SetRecognizableGestures(GestureSettings.NavigationRailsX |
                                                                     GestureSettings.NavigationRailsY |
                                                                     GestureSettings.NavigationRailsZ);
             }
             else
             {
-                navigationGestureRecognizer.SetRecognizableGestures(GestureSettings.NavigationX |
+                NavigationGestureRecognizer.SetRecognizableGestures(GestureSettings.NavigationX |
                                                                     GestureSettings.NavigationY |
                                                                     GestureSettings.NavigationZ);
             }
+
+            if (RecognizerStart == RecognizerStartBehavior.AutoStart)
+            {
+                GestureRecognizer.StartCapturingGestures();
+                NavigationGestureRecognizer.StartCapturingGestures();
+            }
 #endif
         }
 
-        protected virtual void OnDestroy()
+        protected virtual void OnDisable()
         {
 #if UNITY_WSA
-            InteractionManager.InteractionSourceUpdated -= InteractionManager_InteractionSourceUpdated;
+            StopGestureRecognizer();
```

Comment:
This (and StartGestureRecognizer) were renamed to Start/StopGestureRecognizers(). Looks like that's being ignored in the change.

Reply:
Yeah it was kept the same to prevent breaking changes when people updated their projects.

Reply:
How do you decide a "good" breaking change? This entire PR was a breaking change. This is such a minor one, it would be easily updated.

Reply:
Fixed in 1158

Reply:
I'll revert this one.

Reply:
I was mainly worried about current projects updating to the new version, and limiting the number of breaking changes required to update.